docs/HOW_TO_DEPLOY_0_15.md

```diff
@@ -0,0 +1,10 @@
+# KEEP IN MIND WHEN DEPLOYING 0.15
+
+Take a look to the Changelog https://github.com/decidim/decidim/blob/0.15-stable/CHANGELOG.md to specific actions. Instead of run the "Upgrade notes" on Changelog 0.15, execute next queries to prevent some crashes"
```

Comment:
I don't understand this paragraph. You say to check the changelog but then you say to ignore it and instead run "next queries", but where are the "next queries"?

Reply:
Ups, sorry! It was a copy paste from another document... It's fixed now